Orange/classification/logistic_regression.py

```diff
@@ -36,7 +36,7 @@ def coefficients(self):
 class LogisticRegressionLearner(SklLearner, _FeatureScorerMixin):
     __wraps__ = skl_linear_model.LogisticRegression
     __returns__ = LogisticRegressionClassifier
-    name = 'logreg'
+    name = 'logit'
```

Comment:
Number of results for
https://www.google.si/search?num=2&q=logit
vs.
https://www.google.si/search?num=2&q=logreg

Been bugging me for a while.